helm_deploy/postgres/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
As the django chart now uses the main postgres chart as a dependency we don't need this chart at all and it can be deleted.

Reply:
Removed under d5210c3.